src/WebServer.ino

```diff
@@ -5653,7 +5653,7 @@ void handle_sysvars() {
 
   addTableSeparator(F("Custom Variables"), 3, 3);
   for (byte i = 0; i < CUSTOM_VARS_MAX; ++i) {
-    addSysVar_html(String(F("%customvar"))+toString(i+1,0)+String(F("%")));
+    addSysVar_html(String(F("%v"))+toString(i+1,0)+String(F("%")));
   }
```

Comment:
Here you don't use the same "2" for decimals. I don't agree on it, but just wonder why it differs.

Reply:
:)